src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -353,56 +318,160 @@ ApiGlobe.prototype.createSceneGlobe = function createSceneGlobe(coordCarto, view
     var gLDebug = false; // true to support GLInspector addon
     var debugMode = false;
 
-    var ellipsoid = new Ellipsoid({
+    this.ellipsoid = new Ellipsoid({
         x: 6378137,
         y: 6356752.3142451793,
         z: 6378137,
     });
 
     var coordinate = new GeoCoordinate().copy(coordCarto, UNIT.DEGREE);
+    // TODO: use GeoCoordinate conversion instead
+    var positionCamera = this.ellipsoid.cartographicToCartesian(coordinate);
+    this.scene = new Scene(viewerDiv, debugMode, gLDebug);
+    this.scene.camera.setPosition(positionCamera);
 
-    this.scene = Scene(coordinate, ellipsoid, viewerDiv, debugMode, gLDebug);
-
-    var map = new Globe(ellipsoid, gLDebug);
-
-    this.scene.add(map);
-
-    // Register all providers
-    var wmtsProvider = new WMTS_Provider({
-        support: map.gLDebug,
-    });
-
-    this.scene.scheduler.addProtocolProvider('wmts', wmtsProvider);
-    this.scene.scheduler.addProtocolProvider('wmtsc', wmtsProvider);
-    this.scene.scheduler.addProtocolProvider('tile', new TileProvider(ellipsoid));
-    this.scene.scheduler.addProtocolProvider('wms', new WMS_Provider({ support: map.gLDebug }));
+    this.scene.initDefaultProviders(this.scene);
 
     this.sceneLoadedDeferred = defer();
     this.addEventListener('globe-loaded', () => {
         this.sceneLoadedDeferred.resolve();
         this.sceneLoadedDeferred = defer();
     });
 
-    return this.scene;
+    // init globe layer with default parameter
+    const wgs84TileLayer = {
+        // layer base options
+        protocol: 'tile',
+        id: globeLayerId,
+        preUpdate: (context, layer) => preGlobeUpdate(context, layer),
+        update: (context, layer, node) => updateTreeLayer(context, layer, node),
+        // options for 'tile' protocol
+        nodeType: TileMesh,
+        builder: new BuilderEllipsoidTile(this.ellipsoid, new Projection()),
+        // options for tree-based layer
+        initLevel0Nodes: initTiledGeometryLayer(),
+        processNode: processTiledGeometryNode,
+        // options for tilegeometry
+        maxLevel: 18,
+        schemeTile: globeSchemeTileWMTS(globeSchemeTile1),
+        cullingTest: globeCulling,
+        initNewNode,
+        mustSubdivide: globeSubdivisionControl,
+        // options for globe
+        ellipsoid: this.ellipsoid,
+    };
+
+    this.addGeometryLayer(wgs84TileLayer);
+
+    const atmosphere = new Atmosphere(this.ellipsoid);
```

Comment:
instance atmosphere here, is to get out all the specificities of the globe, isn't?